tests/unittests/GraphTest.cpp

```diff
@@ -1672,3 +1672,59 @@ TEST(Graph, GetOutputSaveTest) {
   EXPECT_NE(nullptr, FoundNode);
   EXPECT_EQ(SN2, FoundNode);
 }
+
+/// Check if dump functions work for Node, Function and Module.
+TEST(Graph, testDumpStructure) {
+  Module MD;
+  Function *F = MD.createFunction("F");
+  IRFunction M(F);
+  PlaceholderBindings bindings;
+  Node *K = MD.createPlaceholder(ElemKind::FloatTy, {4, 320, 200, 100, 3},
+                                 "input", true);
+  // Test Node
+  std::string storageN1;
+  llvm::raw_string_ostream osN1(storageN1);
+  K->dump(osN1);
+  std::string mesN = K->toString();
+  std::string expectMes = "Placeholder\nname : \"input\"\noutput : float<4 x "
```

Comment:
Nit: It may look nicer if you'd use raw string literals. See
https://en.cppreference.com/w/cpp/language/string_literal and https://stackoverflow.com/questions/20508534/c-multiline-string-raw-literal/20508617. Try it out and if it looks better - use it and update all such places in your tests!

Reply:
> Nit: It may look nicer if you'd use raw string literals. See
> https://en.cppreference.com/w/cpp/language/string_literal and https://stackoverflow.com/questions/20508534/c-multiline-string-raw-literal/20508617. Try it out and if it looks better - use it and update all such places in your tests!

OK, thanks. Now it looks the same as what the terminal shows.